samza-test/src/test/java/org/apache/samza/test/framework/StreamApplicationIntegrationTest.java

```diff
@@ -135,10 +135,10 @@ public void describe(StreamApplicationDescriptor appDescriptor) {
               KVSerde.of(new IntegerSerde(), new TestTableData.ProfileJsonSerde())));
 
       KafkaSystemDescriptor ksd = new KafkaSystemDescriptor("test");
-      KafkaInputDescriptor<TestTableData.Profile> profileISD = ksd.getInputDescriptor("Profile", new NoOpSerde<>());
+      KafkaInputDescriptor<TestTableData.Profile> profileISD = ksd.getInputDescriptor("Profile", new JsonSerdeV2<>());
```

Comment:
Set a KVSerde.of() so that this can test key serde usage as well.

Reply:
sure